sdk/appconfiguration/app-configuration/src/internal/helpers.ts

```diff
@@ -230,6 +230,8 @@ export function transformKeyValue<T>(kvp: T & KeyValue): T & ConfigurationSettin
 
   delete setting.locked;
 
+  if (kvp.label) setting.label = kvp.label;
```

Comment:
AFAIU this line won't work as expected, i.e. making setting.label undefined.

If `kvp.label` is null, `setting.label` is already assigned the value null in line 227, because of the spread operator is used.